spec/models/partnership_spec.rb

```diff
@@ -25,7 +25,8 @@
     user = profile.user
     user.update!(updated_at: 2.weeks.ago)
 
-    partnership.touch
+    partnership.update!(updated_at: 2.weeks.ago)
```

Comment:
random fyi, touch can take arguments eg
```
partnership.touch(time: 2.weeks.ago)
```
don't think you need to change though

Reply:
Good to know!